assign4.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs');
+var file1Content= ` file 1 content goes 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
+  1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 
+  1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 `;
+fs.writeFile("file1.txt",file1Content,function(err,data) {
+  if(err)
+    throw err;
+  else
+    console.log("file 1 content");
+});
+
+fs.readFile("file1.txt",(err,data) => {
+  if(err)
+    throw err;
+  else
```

Comment:
throw error breaks the execution flow hence no need to write `else` explicilty. If there's an error line written below won't get executed.